cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/CgmesConformity3ConversionTest.java

```diff
@@ -114,22 +113,19 @@ void microGridBaseCaseBEMergingViewNL() {
         // Both networks have the same number of dangling lines
         assertEquals(nDlBE, nDlNL);
 
-        Network n = MergingView.create("be-nl", "CGMES");
-        n.merge(nl, be);
+        be.merge(nl);
```

Comment:
We see here that we have an equivalence now between the merging view and the destructive merge for equipments.

Reply:
I think this test can be merged with the test above `microGridBaseCaseBEMergedWithNL`, they check the same thing now...